packages/devtools/lib/src/ui/icons.dart

```diff
@@ -132,13 +135,13 @@ class FlutterIcons {
   static const Icon history = UrlIcon('/icons/history.svg');
 
   static const UrlIcon pause_black_2x =
-      UrlIcon('/icons/general/pause_black@2x.png');
+      UrlIcon('/icons/general/pause_black@2x.png', invertDark: true);
   static const UrlIcon pause_black_disabled_2x =
```

Comment:
add a TODO to remove the invertDark option from these 4 buttons as it is a bit odd.

Reply:
Added a TODO, though do you mean 3?

I was a bit unsure about these - we don't use them currently, but if you do put a white pause button on a dark button in the light theme, you'd probably want it inverted to?

That said, if we do remove invert from the white one, we could ditch the "primary" version an just use the white one.

I'll land this as-is (with the TODO linking here), but lmk what you think is best and I'll change it.